### PR TITLE
Feature/sidebar list fix

### PR DESCRIPTION
**Pull request by mlonn**
Added boilerplate components for links in sidebar and fixed sidebar dropdowns

### CODE REVIEW

File: frontend/web/src/app/components/sidemenu/sidemenu.component.html [hunk 1]
```diff
@@ -14,30 +14,30 @@
...
           </li>
         </ul>
       </li>
 
-      <li class="dropdown">
-        <a href="#" class="dropdown-toggle" data-toggle="dropdown">Prints
-          <span class="caret"></span>
+      <li>
+        <a href="#prints" class="collapsed" data-toggle="collapse">Prints
+          <i class="glyphicon glyphicon-menu-down"></i>
```
Comment (chraker) [inline frontend/web/src/app/components/sidemenu/sidemenu.component.html:33]: MINOR Replace this <i> tag by <em>. [rule](https://sonarcloud.io/coding_rules#rule_key=Web%3ABoldAndItalicTagsCheck)

File: frontend/web/src/app/components/sidemenu/sidemenu.component.html [hunk 2]
```diff
@@ -14,30 +14,30 @@
       <li>
         <a routerLink="orders" class="nav-link" href="#" routerLinkActive="active">Orders</a>
       </li>
-      <li class="dropdown">
-        <a href="#" class="dropdown-toggle" data-toggle="dropdown">Parts
-          <span class="caret"></span>
+      <li>
+        <a href="#parts" class="collapsed" data-toggle="collapse">Parts
+          <i class="glyphicon glyphicon-menu-down"></i>
```
Comment (chraker) [inline frontend/web/src/app/components/sidemenu/sidemenu.component.html:19]: MINOR Replace this <i> tag by <em>. [rule](https://sonarcloud.io/coding_rules#rule_key=Web%3ABoldAndItalicTagsCheck)

**Comment by chraker** [comment]
SonarQube analysis reported 2 issues
* ![MINOR][MINOR] 2 minor

Watch the comments in this conversation to review them.

[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

**Review by eineving — APPROVED** [review]
Looks like it is minor faults

**Pull request merged**